sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <full-page-container>
+    <h2 class="color-blue">Search Institution</h2>
+    <v-row class="mt-5">
+      <v-col><label for="legalName">Legal Name</label></v-col>
```

Comment:
Labels and inputs are supposed to be grouped together. Can you adjust it? The search button also seems to be lost in the screen on the right. Can you try to adjust it?

![image](https://user-images.githubusercontent.com/61259237/142261570-04459dfa-77e8-43d0-b3bb-618a124b5321.png)

There an simular example on _SupportingInformation.vue_. Please take a look if makes sense to follow it.